### DISCUSSION TITLE
Readme says xls is not supported but I was able to use it?

### DISCUSSION THREAD

**Discussion by mauro-rizzi-DSP** [pjfanning/excel-streaming-reader#229] [General]
Hello, I don't know that mutch about the internal format of excel files but I accidentally tried several different xls files with this streaming reader and they were all processed (after I spent days setting up my app to use different libs depending on the file extension). Does this library just default to the regular poi implementation for those files automatically? or are these xls files i'm mentioning somehow xlsx disguised as xls? having it default to the standard poi implementation would solve a ton of issues for me but I wanted confirmation

**Comment by pjfanning** [comment]
https://github.com/pjfanning/excel-streaming-reader?tab=readme-ov-file#implementation-details

Docs say: This library will ONLY work with XLSX files. The older XLS format is not capable of being streamed.

I would appreciate if you read the docs.

**Reply by mauro-rizzi-DSP** [reply]
I read the readme and spent a couple of days working around having different implementations for xlsx and xls but then I was able to read a xls file using this library by accident hence the question, i'll take your RTFM answer as a "it's not supposed to, no"

Image proof:
![xls test](https://github.com/pjfanning/excel-streaming-reader/assets/111378929/84da446b-4e10-41bc-aa81-f082fe6bbc09)

EDIT: I redacted the contents as you can see but it read the file just fine
